user-interface/README.md

```diff
@@ -0,0 +1,68 @@
+# User interface acceptance criteria
+This folder contain descriptions of things that users do when interacting with Vega. The information they need and why they are doing them. These can be referenced in testing and used as input for building new interfaces.
```

Comment:
I feel like there's a mix up here of generic requirements and project specific ones, especially for the desktop wallet project and Console. Would be good to know what the plan for this is and agree how we manage:

- the responsibilities individual projects have to deliver core "Vega wide" requirements vs. 
- the requirmeents for a specific project/UI

I don't feel this is clear currently

Reply:
Responded above.

I guess if not here (or in a folder nearby), where, for product/project specific reqs?